x-pack/plugins/siem/public/components/page/network/kpi_network/index.tsx

```diff
@@ -19,86 +21,91 @@ interface KpiNetworkProps {
   loading: boolean;
 }
 
-const fieldTitleMapping: Readonly<CardItems[]> = [
+const fieldTitleMapping: Readonly<StatItems[]> = [
   {
     fields: [
       {
         key: 'networkEvents',
-        description: i18n.NETWORK_EVENTS,
         value: null,
       },
     ],
+    description: i18n.NETWORK_EVENTS,
   },
   {
     fields: [
       {
         key: 'uniqueFlowId',
-        description: i18n.UNIQUE_ID,
         value: null,
       },
     ],
+    description: i18n.UNIQUE_ID,
   },
   {
     fields: [
       {
         key: 'activeAgents',
-        description: i18n.ACTIVE_AGENTS,
         value: null,
       },
     ],
+    description: i18n.ACTIVE_AGENTS,
   },
   {
     fields: [
       {
         key: 'uniqueSourcePrivateIps',
-        description: i18n.UNIQUE_SOURCE_PRIVATE_IPS,
         value: null,
       },
     ],
+    description: i18n.UNIQUE_SOURCE_PRIVATE_IPS,
   },
   {
     fields: [
       {
         key: 'uniqueDestinationPrivateIps',
-        description: i18n.UNIQUE_DESTINATION_PRIVATE_IPS,
         value: null,
       },
     ],
+    description: i18n.UNIQUE_DESTINATION_PRIVATE_IPS,
   },
   {
     fields: [
       {
         key: 'dnsQueries',
-        description: i18n.DNS_QUERIES,
         value: null,
       },
     ],
+    description: i18n.DNS_QUERIES,
   },
   {
     fields: [
       {
         key: 'tlsHandshakes',
-        description: i18n.TLS_HANDSHAKES,
         value: null,
       },
     ],
+    description: i18n.TLS_HANDSHAKES,
   },
 ];
 
 export const KpiNetworkComponent = pure<KpiNetworkProps>(({ data, loading }) => {
-  return (
+  return loading ? (
+    <EuiFlexGroup style={{ minHeight: 86 }} justifyContent="center" alignItems="center">
```

Comment:
Throughout the code we go for a Styled approach (for now) rather than inline where possible. So this would be:

```ts
const FlexGroup = styled(EuiFlexGroup)`
  margin-height: 86px;
`;
```